Assets/Python/GreatPeople.py

```diff
@@ -65,7 +65,7 @@ def getName(unit):
 	
 	return utils.getRandomEntry(lAvailableNames)
 	
-def onGreatPersonBorn(unit, iPlayer, city):
+def onGreatPersonBorn(unit, iPlayer, city, bWB = False):
```

Comment:
Can you rename the new argument to something that is meaningful to the method itself? I would prefer something like `bAnnounceBirth = True`.